HW_3/form.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Length, EqualTo, Email
+
+class RegistrationForm(FlaskForm):
+    lastname = StringField('Lastname', validators=[DataRequired()])
+    firstname = StringField('Firstname', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
+    password = PasswordField('Password', validators=[DataRequired(), Length(min=6)])
+    confirm_password = PasswordField('Confirm Password', validators=[DataRequired(), EqualTo('password')])
```

Comment:
Если добавить поле `SubmitField`.
```suggestion
    confirm_password = PasswordField('Confirm Password', validators=[DataRequired(), EqualTo('password')])
    submit - SubmitField('Зарегистрироваться')
```